src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int firstSide;
+    private final int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, String color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        setColor(color);
+        obtainTheArea();
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables and pass them on for better code readability. Also, avoid calling non-constructor methods like `obtainTheArea()` from within the constructor; it's better to handle such operations outside of it.